### PR TITLE
Feature/similar components mobile/desktop update

### PR DESCRIPTION
**Pull request by DStukalo**

### CODE REVIEW

**Comment by MaksVibe** [comment]
We are working from "client" directory. You've placed project at the root. Please place your changes in "client" directory for us to make the merge. Thanks

File: client/components/Button/Button.styles.ts [hunk 1]
```diff
@@ -9,10 +9,21 @@ export const buttonStyle = css`
 	border-radius: 4px;
 	color: ${theme.colors.mainTitle};
 	background-color: ${theme.colors.mainText};
-	border:none;
+	border: none;
```
Comment (MaksVibe) [inline client/components/Button/Button.styles.ts]: pls, fix broder to   `  border: 1px solid transparent;`

File: client/components/Label/Label.tsx [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
-import { labelStyle } from "./Label.styles";
+'use client';
+import { ReactNode } from 'react';
+import { labelStyle, labelErrorStyle, labelSuccessStyle } from './Label.styles';
 
 interface labelProps {
 	forHTML: string;
 	children: ReactNode | string;
+	isError: boolean;
+	isSuccess: boolean;
```
Comment (MaksVibe) [inline client/components/Label/Label.tsx]: `isError` and `isSuccess` are optional props, fix ps

File: client/components/Title/Title.tsx [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+import { titleStyle } from './Title.styles';
 
 interface TitleProps {
 	children: string;
 }
 
 const Title = ({ children }: TitleProps) => {
-	return <h2 css={titleStyle}>{children}</h2>;
+	return <h1 css={titleStyle}>{children}</h1>;
 };
```
Comment (MaksVibe) [inline client/components/Title/Title.tsx:28]: Set h2 for modal title pls as shown below

Screenshot 2023-07-10 at 13 56 21

**Comment by MaksVibe** [comment]
Rename button component pls
Screenshot 2023-07-10 at 13 59 03

**Review by MaksVibe — CHANGES_REQUESTED** [review]

File: client/components/Label/Label.tsx (hunk 2 above)
Comment (DStukalo) [reply client/components/Label/Label.tsx]: I think we need to directly set our component state, I'll rework it to reduce the number of props passed

File: client/components/Title/Title.tsx (hunk 3 above)
Comment (DStukalo) [reply client/components/Title/Title.tsx:28]: But we don't have h1 in our landing

File: client/components/Label/Label.tsx (hunk 2 above)
Comment (MaksVibe) [reply client/components/Label/Label.tsx]: Then, why those props are supposed to be here?

**Review by MaksVibe — CHANGES_REQUESTED** [review]

**Review by MaksVibe — APPROVED** [review]

**Pull request merged**